src/main/kotlin/mods/eln/mechanical/Generator.kt

```diff
@@ -72,13 +73,12 @@ class GeneratorDescriptor(
     ).requireNoNulls()
 
     override fun addInformation(stack: ItemStack, player: EntityPlayer, list: MutableList<String>, par4: Boolean) {
-        list.add("Converts mechanical energy into ")
-        list.add("electricity, or (badly) vice versa.")
-        list.add("Nominal usage ->")
-        list.add(Utils.plotVolt("  Voltage out: ", nominalU.toDouble()))
-        list.add(Utils.plotPower("  Power out: ", nominalP.toDouble()))
-        list.add(Utils.plotRads("  Rads: ", nominalRads.toDouble()))
-        list.add(Utils.plotRads("Max rads:  ", absoluteMaximumShaftSpeed))
+        list.add(I18N.tr("Converts mechanical energy into electricity, or (badly) vice versa."))
+        list.add(I18N.tr("Nominal usage ->"))
+        list.add("  "+I18N.tr("Voltage out: %1$",Utils.plotVolt("", nominalU.toDouble())))
+        list.add("  "+I18N.tr("Power out: %1$",Utils.plotPower("", nominalP.toDouble())))
+        list.add("  "+I18N.tr("Rads: %1$",Utils.plotRads("", nominalRads.toDouble())))
+        list.add(I18N.tr("Max rads:  %1$",Utils.plotRads("", absoluteMaximumShaftSpeed)))
```

Comment:
Hmm. Is it possible to put the translation where you have `""` in the `plot*()` functions? Maybe that doesn't make sense but we should consider making this less boilerplate-y.

Reply:
I will modify similar content.